Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Beckn/ACL/OnSearch.hs

```diff
@@ -61,12 +63,7 @@ mkOnSearchMessage res@DSearch.DSearchRes {..} = do
             rides_completed = 0, --FIXME
             rides_confirmed = 0 --FIXME
           }
-      payment =
-        OS.Payment
-          { collected_by = "BPP",
-            _type = OS.ON_FULFILLMENT,
-            time = OS.TimeDuration "P2A" -- FIXME: what is this?
-          }
+      payments = mkPayment <$> paymentMethodsInfo
```

Comment:
can we handle backward compatibility here also, as older version of BAP will expect `payment` to be there.